.devcontainer/docker-compose-vscode.yml

```diff
@@ -12,19 +12,20 @@ services:
     # Overrides default command so things don't shut down after the process ends.
     command: /bin/sh -c "while sleep 1000; do :; done"  
     environment:
-      - MYSQL_PASSWORD=/run/secrets/mysql_password
-      - SIMULATION_MODE=true
-      - SIMULATION_IP=127.0.0.1
-      - SIMULATION_REGISTRATION_PORT=6767
-      - LOCAL_IP=127.0.0.1
-      - TIME_SYNC_TOPIC=time_sync
-      - TIME_SYNC_PORT=7575
-      - SIM_V2X_PORT=5757
-      - SIM_INTERACTION_PORT=7576
-      - V2X_PORT=8686
-      - INFRASTRUCTURE_ID=rsu_<J2735 MAP MESSAGE INTERSECTION ID>
-      - INFRASTRUCTURE_NAME=<RSU_NAME>
-      - SENSOR_JSON_FILE_PATH=/var/www/plugins/MAP/sensors.json
+      MYSQL_PASSWORD: /run/secrets/mysql_password
+      SIMULATION_MODE: false
+      # Environment variables only requires in simulation 
```

Comment:
Up to you - would it be easier to leave commented in for someone to just change simulation_mode or do you prefer to leave these commented out for developers to comment back in?